planning/behavior_path_planner/autoware_behavior_path_lane_change_module/test/test_lane_change_scene.cpp

```diff
@@ -0,0 +1,228 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware/behavior_path_lane_change_module/manager.hpp"
+#include "autoware/behavior_path_lane_change_module/scene.hpp"
+#include "autoware/behavior_path_lane_change_module/utils/data_structs.hpp"
+#include "autoware/behavior_path_planner_common/data_manager.hpp"
+#include "autoware_test_utils/autoware_test_utils.hpp"
+#include "autoware_test_utils/mock_data_parser.hpp"
+
+#include <ament_index_cpp/get_package_share_directory.hpp>
+
+#include <gtest/gtest.h>
+
+#include <memory>
+
+using autoware::behavior_path_planner::LaneChangeModuleManager;
+using autoware::behavior_path_planner::LaneChangeModuleType;
+using autoware::behavior_path_planner::NormalLaneChange;
+using autoware::behavior_path_planner::PlannerData;
+using autoware::behavior_path_planner::lane_change::CommonDataPtr;
+using autoware::behavior_path_planner::lane_change::LCParamPtr;
+using autoware::behavior_path_planner::lane_change::RouteHandlerPtr;
+using autoware::route_handler::Direction;
+using autoware::route_handler::RouteHandler;
+using autoware::test_utils::get_absolute_path_to_config;
+using autoware::test_utils::get_absolute_path_to_lanelet_map;
+using autoware::test_utils::get_absolute_path_to_route;
+using autoware_map_msgs::msg::LaneletMapBin;
+using geometry_msgs::msg::Pose;
+using tier4_planning_msgs::msg::PathWithLaneId;
+
+class TestNormalLaneChange : public ::testing::Test
+{
+public:
+  void SetUp() override
+  {
+    rclcpp::init(0, nullptr);
+    init_param();
+    init_module();
+  }
+
+  void init_param()
+  {
+    auto node_options = init_node_options();
+    auto node = rclcpp::Node(name, node_options);
+    planner_data_->init_parameters(node);
+    lc_param_ptr_ = LaneChangeModuleManager::set_params(&node, node.get_name());
+    planner_data_->route_handler = init_route_handler();
+
+    ego_pose_ = autoware::test_utils::createPose(-50.0, 1.75, 0.0, 0.0, 0.0, 0.0);
+    planner_data_->self_odometry = set_odometry(ego_pose_);
+    planner_data_->dynamic_object =
+      std::make_shared<autoware_perception_msgs::msg::PredictedObjects>();
+  }
+
+  void init_module()
+  {
+    normal_lane_change_ =
+      std::make_shared<NormalLaneChange>(lc_param_ptr_, lc_type_, lc_direction_);
+    normal_lane_change_->setData(planner_data_);
+    set_previous_approved_path();
+  }
+
+  [[nodiscard]] const CommonDataPtr & get_common_data_ptr() const
+  {
+    return normal_lane_change_->common_data_ptr_;
+  }
+
+  [[nodiscard]] rclcpp::NodeOptions init_node_options() const
```

Comment:
`get_node_options` seems more appropriate here

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/9090/commits/0e362e625dead16c68378d8254b438b9e79c2799